### ISSUE TITLE
CrashUtilities unloads party chunks when 1 party member is offline for long enough

### ISSUE THREAD

**Issue by Clawthorn** [Darkere/CrashUtilities#24]
I suppose I should mention we're playing the **ATM6 Modpack** so the feature is **enabled by default**.

CrashUtilities is unloading chunks of entire parties (regardless of who is the owner of that party) when one of the members in that party have been offline for the time set in `crashutilities-server.toml`.

**How to reproduce:**
1. Join a private server with a friend.
2. Start a party using FTBTeams, and invite your friend to that party.
3. Claim and load a few chunks. The claim will be owned by the party (and you own that party).
4. Ask your friend to log out for X days, while you keep playing.
5. Log out yourself for a little while (less than 12 hours)
6. Server console will say "Unloading chunks for players not online for X days: YourFriendName"
7. Log back in, all the chunkloads by the party has been removed

**Expected result:**
CrashUtilities should see that the chunkloading is made by a party, and that the party is still "active" on the server because at least one of the members are still actively playing. Thus it should NOT unload chunks belonging to the party.

**Actual result:**
CrashUtilities unloads all chunks of a party, and keeps repetedly unloading them, because one of the members of the party has not been online for X days.

**Comment by gdharron** [comment]
any update on this? I also have this issue and I think CU should only check if the owner or any party member has been active.

**Comment by Darkere** [comment]
CU does not actually interact in any direct way with FTB chunks. Just runs some minecraft commands if it sees a player is offline for a while. 

This was more or less a stop gap feature before FTBChunks implemented disabling offline chunkloading. 

Since that has happened I would rather just remove this feature.

**Comment by gdharron** [comment]
It's a great feature imo on medium/large servers. I still want offline chunk loading but if someone goes away for over 48 hours I have it set to unload their chunks to save resources which I don't think you can do with FTB chunks

**Comment by zediious** [comment]
Would absolutely love a fix for this, I was ecstatic to be able to time-limit offline loading ahead of the actual FTB Chunks feature.

**Comment by KuroPeach** [comment]
yep, Same problem playing ATM6SKY  @https://github.com/AllTheMods
And when it unloads the block, it has a chance to cause the server to freeze and be killed by the watchdog
https://paste.ubuntu.com/p/4CBvkhjWrQ/

**Comment by sprocketaudio** [comment]
reading the above I guess this won't get fixed but I'll stil say it would be great if somehow it could! I rely on the unloading of chunks after x has been offline for y days as a server admin, but if it could somehow interface with ftbteams to get team members of each team, then check that againt who has been offline, and if it finds a member of a team has been online within the set time frame, ignore chunk unloading for all members of that team.
